package.json

```diff
@@ -39,6 +39,7 @@
     "remark-lint-no-unused-definitions": "1.x.x",
     "remember-bower": "0.1.x",
     "sanctuary-style": "1.1.x",
+    "spdx-expression-parse": "3.0.x",
```

Comment:
[`spdx-expression-parse`][1] is used by npm itself, so should be solid.


[1]: https://www.npmjs.com/package/spdx-expression-parse